pb/xcheck.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package pb;
+
+import "google/api/annotations.proto";
+import "xchain.proto";
+
+service Xcheck {
+    rpc ComplianceCheck(TxStatus) returns (ComplianceCheckResponse);
```

Comment:
this is a client rpc proto, it's better keep the proto in cmd/cli.
pb folder is more like server-side proto.

Reply:
Yes, your advice is better and i will fix it.